_includes/component.html

```diff
@@ -2,7 +2,8 @@
 {% assign componentKey = componentName | slugify %}
 {% assign settingsKey = componentKey %}
 {% assign component = site.data.accessibility-tests[componentKey] %}
-{% assign issues = site.data.issues[componentKey] %}
+{% assign known_issues_key = page.known_issues_key | default: page.title | slugify %}
```

Comment:
This is working great for me but I'm curious if there would ever be a scenario where we would want to include multiple known issue sections.

Originally I was wondering if there were specific date range picker issues **and** general date picker issues and if we'd want to include both. Creating a date range picker issue yml file showed that only the `known_issues_key` would listed on the page.

Another potential use case would be general form issues that we might want to include on multiple component pages.

This is probably out of scope for this work but I wanted to run it by you and see what you were thinking!

Reply:
I ultimately came to the conclusion that sharing known issues across pages will be fairly rare. The exceptions might be button and button group, maybe icon/icon list, and maybe forms. It isn't clear to me what these pages might need (or if they'll need anything at all), so I went ahead and just built a solution for just this particular use case. I figured we can cross the bridge if/when we get to it. 

Happy to discuss other solutions if you want!